pilot/helpers/Project.py

```diff
@@ -408,6 +408,7 @@ def save_file(self, data):
             inputs_required = self.find_input_required_lines(data['content'])
             for line_number, line_content in inputs_required:
                 user_input = None
+                print('', type='verbose', category='human-intervention')
                 print(color_yellow_bold(f'Input required on line {line_number}:\n{line_content}') + '\n')
```

Comment:
instead of having extra print you can just do ` print(color_yellow_bold(f'Input required on line {line_number}:\n{line_content}') + '\n', type='verbose', category='human-intervention')`